src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -1565,7 +1580,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Debug.Assert(initializer.Syntax.Span.Start > initializers.Last().Syntax.Span.Start)
             End If
 
+            initializer.PrecedingInitializersLength = aggregateSyntaxLength
```

Comment:
Can PrecedingInitializersLength be a ReadOnly field? (Perhaps create the FieldOrPropertyInitializer here, as in the C# implementation, rather than in the caller.)